Guides/dev-set-up-environment.md

```
@@ -8,20 +8,28 @@
-    - [3.1.2 Step 2: Set up core services](#312-step-2-set-up-core-services)
-  - [3.2. Setting Up a Microservice Processor to Work On](#32-setting-up-a-microservice-processor-to-work-on)
-- [4. Further reading](#4-further-reading)
+- [3. Development environment setup](#3-development-environment-setup)
+- [4. Microprocessor setup instructions](#4-microprocessor-setup-instructions)
+  - [4.1. Preparation](#41-preparation)
+    - [4.1.1 Step 1: Setting up GitHub Token Locally](#411-step-1-setting-up-github-token-locally)
+    - [4.1.2 Step 2: Set up core services](#412-step-2-set-up-core-services)
+    - [4.1.3 Step 3: Set up optional services](#413-step-3-set-up-optional-services)
+  - [4.2. Setting Up a Tazama Processor to Work On](#42-setting-up-a-tazama-processor-to-work-on)
+    - [4.2.1 Setting up the rule module](#421-setting-up-the-rule-module)
+    - [4.2.2 Setting up the rule executer](#422-setting-up-the-rule-executer)
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)